pages/linux/genie.md

```diff
@@ -0,0 +1,17 @@
+# genie
+
+> Set up and use a "bottle" namespace to run systemd under WSL (Windows Subsystem for Linux).
+> To run these from Windows rather than an already-running distribution, precede them with `wsl`.
+> Further information available at: <https://github.com/arkane-systems/genie>.
```

Comment:
See the [style guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md)

```suggestion
> More information: <https://github.com/arkane-systems/genie>.
```